.appveyor/appveyor.yml

```diff
@@ -2,7 +2,7 @@ version: '{branch}.{build}'
 branches:
   only:
   - v1
-clone_folder: c:\gopath\src\gopkg.in\gcfg.v1
+clone_folder: c:\gopath\src\gopkg.in\please-build\gcfg.v1
```

Comment:
What's thing thing? we can probably get rid of it.

Reply:
No idea. I'll remove it if we are not using it